src/auth-service/services/mailer.js

```diff
@@ -84,16 +84,18 @@ const mailer = {
         }
       }
 
-      const categoryNameWithFirstLetterCapital =
-        category.charAt(0).toUpperCase() + category.slice(1);
+      /**
+       * const categoryNameWithFirstLetterCapital = category.charAt(0).toUpperCase() + category.slice(1);
+       * const subject = `Welcome to AirQo, for ${categoryNameWithFirstLetterCapital}`
+       */
 
       const mailOptionsForAirQo = {
         to: `${email}`,
         from: {
           name: constants.EMAIL_NAME,
           address: constants.EMAIL,
         },
-        subject: `Welcome to AirQo, for ${categoryNameWithFirstLetterCapital}`,
+        subject: `Welcome to AirQo`,
```

Comment:
As per feedback from @ebainomu , just changing the email subject for inquiry emails.